test/unit/test_skos_history_wrapper.py

```diff
@@ -0,0 +1,118 @@
+"""
+test_skos_history_wrapper.py
+Date: 06/07/2020
+Author: Mihai Coșleț
+Email: coslet.mihai@gmail.com
+"""
+from filecmp import cmp
+from pathlib import Path
+
+import pytest
+
+from rdf_differ.skos_history_wrapper import SKOSHistoryRunner, SKOSHistoryFolderSetUp
+from rdf_differ.utils import dir_exists
+
+
+def helper_endpoint_mock(monkeypatch):
+    monkeypatch.setenv('ENDPOINT', 'http://test.point')
```

Comment:
Why not just monkeypath the env directly in the test? It's more isolated and also clear when you look at the test as you instantly see that you patch an env var.